.github/workflows/CI.yml

```diff
@@ -17,6 +18,18 @@ jobs:
           - beta
           - nightly
           - "1.48.0"
+        triple:
```

Comment:
Not a triple anymore :)
I would use something more generic as "projects" or "crates"

Reply:
First, making it work, then cleanup! :laughing: But yes, you're absolutely right.